sdk/tests/ED25519PublicKeyTest.cc

```diff
@@ -107,39 +107,39 @@ TEST_F(ED25519PublicKeyTest, VerifySignatureAgainstModifiedBytes)
   const std::vector<unsigned char> signature = getTestPrivateKey()->sign({ 0x1, 0x2, 0x3 });
   const std::vector<unsigned char> modifiedBytes = { 0x1, 0x2, 0x3, 0x4 };
 
-  EXPECT_FALSE(getTestPublicKeyFromPrivate()->verifySignature(signature, modifiedBytes));
-  EXPECT_FALSE(getTestPublicKeyFromString()->verifySignature(signature, modifiedBytes));
-  EXPECT_FALSE(getTestPublicKeyFromProtobuf()->verifySignature(signature, modifiedBytes));
+  EXPECT_ANY_THROW(getTestPublicKeyFromPrivate()->verifySignature(signature, modifiedBytes));
+  EXPECT_ANY_THROW(getTestPublicKeyFromString()->verifySignature(signature, modifiedBytes));
+  EXPECT_ANY_THROW(getTestPublicKeyFromProtobuf()->verifySignature(signature, modifiedBytes));
```

Comment:
Since only a `std::runtime_error` is expected to be thrown here, you could do `EXPECT_THROW(getTestPublicKeyFrom<variant>()->verifySignature(signature, modifiedBytes), std::runtime_error);` or something to that effect instead. This goes for all updated test assertions in this PR.

Reply:
All of the tests have been updated:
 - test `VerifySignatureAgainstModifiedBytes` - in 370451f0a120d5bf95b69150da2e7c5dab62a529;
 - test `VerifyArbitrarySignature` - in 712931257071667967a4a592fb31e0e6394517a2;
 - test `VerifyEmptySignature` - in 1fe2f718d534b9ed622c2b41a316b320e5bb7db7;
 - test `VerifyEmptyMessage` - in d708b8d5ce69ee84751ea3cd2a4d52f5f2dd4b21;